docs/_config.yml

```diff
@@ -1,10 +1,11 @@
 # Site settings
 title: Solace Samples SEMP
+summary: These tutorials get you up to speed sending and receiving messages with Solace technology.
```

Comment:
We need a slightly different summary here since this is SEMP and it's about configuration. So perhaps "These tutorials will get you to up to speed and managing Solace technology."